examples/tv-casting-app/linux/main.cpp

```diff
@@ -114,6 +115,30 @@ HelpOptions helpOptions("tv-casting-app", "Usage: tv-casting-app [options]", "1.
 
 OptionSet * allOptions[] = { &cmdLineOptions, &helpOptions, nullptr };
 
+static void OnBindingAdded(const EmberBindingTableEntry & binding)
+{
+    if (binding.type == EMBER_MULTICAST_BINDING)
+    {
+        ChipLogError(NotSpecified, "Group binding received");
```

Comment:
Why is this LogError? The text does not specify. Text should contain "not yet implemented" or "not supported" and if not supported, a comment stating why (or if not yet implemented, a comment providing details in that case as well)

Reply:
fixed (should have been progress and not error)